### DISCUSSION TITLE
Multi-camera multi cams

### DISCUSSION THREAD

**Discussion by HoleHogMedia** [latenitefilms/BRAWToolbox#151] [Support] [answered]
So what would be the preferred workflow for a multi camera interview where one camera output is BRAW, and the other camera is not a BlackMagic?

**Comment by jefforig** [comment, marked as answer]
I am not sure if this is the best or preferred workflow but when we shoot a multi camera in BRAW, we import the clips using Synchronize clips instead of the multicam.  Then we make a multicam clip with those synched clips and our separate audio.  It seems to work so far.  We just finished a shoot where we mixed Sony footage with BRAW from our BMPC4Ks but we have not synched the clips yet.  I can report back once we start editing that.  But I am pretty confident it will be fine.

**Reply by latenitefilms** [reply]
Yes, this is the best way to do it. You can also use these Synchronised Clips with Sync-N-Link if everything is timecode synced.
